app/views/equipment_models/_table.html.erb

```diff
@@ -1,23 +1,38 @@
-<table id="table_equipment_models" class="datatable-wide table table-striped table-bordered">
+<table id="table_woo" class="datatable table table-striped table-bordered">
   <thead>
   <tr>
-    <th>Name</th>
-    <%= "<th>Type</th>".html_safe unless @category %>
-  <%# these blank TH's necessary for the sort function %>
+    <th class="text-center col-md-2">Name</th>
+    <th class="no_sort text-center" style="vertical-align:middle">Current Status</th>
```

Comment:
We could probably move all of this styling into one of our stylesheets so that you don't have to repeat it on every line, e.g.

```
#table_equipment_models {
  th, td {
    vertical-align: middle;
  }
}
```
(if I'm remembering my SASS syntax correctly)